packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3561,6 +3559,25 @@ class Baseline extends SingleChildRenderObjectWidget {
   }
 }
 
+/// A widget that causes the parent to ignore the [child] for the purposes
+/// of baseline alignment.
+///
+/// See also:
+///
+///  * [Baseline], a widget that positions a child relative to a baseline.
+class IgnoreBaseline extends SingleChildRenderObjectWidget {
+  /// Creates a widget that ignores the child for baseline alignment purposes.
+  const IgnoreBaseline({
+    super.key,
+    super.child,
+  });
+
+  @override
+  RenderIgnoreBaseline createRenderObject(BuildContext context) {
+    return RenderIgnoreBaseline();
+  }
+}
+
```

Comment:
Nit: Too many newlines here.

Reply:
it's a section separator (the next section starts with an all-caps heading), the rest of the file (with one exception i can fix) has the same double blank line before headings.

I don't feel strongly one way or the other if you would prefer I removed the double blank lines everywhere in this file before the headings, what do you think?

Reply:
I say don't worry about it here, maybe if it comes up again we can make this file all 1 line.